pkg/constants/constants.go

```diff
@@ -18,6 +18,12 @@ const (
 	BondDevice = "bond0"
 	// VPNNetworkPrefixSize is the required prefix size for the VPN network.
 	VPNNetworkMask = 96
+
+	//TODO: move to gardener
+	ShootPodNetworkMapped  = "244.0.0.0/8"
+	ShootSvcNetworkMapped  = "243.0.0.0/8"
```

Comment:
Not a big fan of abbreviating if the name is anyway that long, but maybe this goes away soon.